api-client/src/camera/createCapturePreviewImage.ts

```diff
@@ -0,0 +1,20 @@
+import { POST, request } from '../request'
+
+import type {
+  CameraImageSettings,
+  CameraImageSettingsResponse,
+} from '../camera'
+import type { ResponsePromise } from '../request'
+import type { HostConfig } from '../types'
+
+export function createCapturePreviewImage(
+  config: HostConfig,
+  data: CameraImageSettings
+): ResponsePromise<CameraImageSettingsResponse> {
```

Comment:
```suggestion
): ResponsePromise<CameraImageSettingsResponse> {
```
We probably want a `DownloadedImageFileResponse` here that is a `Blob`. This exists in `dataFiles` already, but let's duplicate it in the camera `types.ts` file.